yoeo/detect.py

```diff
@@ -101,7 +101,7 @@ def detect_image(model, image, img_size=416, conf_thres=0.5, nms_thres=0.5):
         detections, segmentations = model(input_img)
         detections = non_max_suppression(detections, conf_thres, nms_thres)
         detections = rescale_boxes(detections[0], img_size, image.shape[:2])
-    return detections.numpy(), segmentations.numpy()
+    return detections.cpu().detach().numpy(), segmentations.cpu().detach().numpy()
```

Comment:
This only needs to be done for the segmentations imo, as the output from the nms is already on the cpu.

Reply:
changed